### ISSUE TITLE
Incompatible with React Bootstrap 1.4.0

### ISSUE THREAD

**Issue by barbalata** [pushtell/react-bootstrap-date-picker#170]
<h1>Table of Contents</h1>

- [Context](#context)
- [Possible Solution](#possible-solution)
- [Your Environment](#your-environment)

## Context

I tried to use the component (latest version) with **React 16.13.1** and **React Bootstrap 1.4.0** and I receive the following error:
```
./node_modules/react-16-bootstrap-date-picker/lib/index.js
Module not found: Can't resolve 'react-bootstrap/lib/Button' in '...\node_modules\react-16-bootstrap-date-picker\lib'
```

## Possible Solution

I see that in version 1.0.0 of React Bootstrap there is no "_lib_" directory and all the modules are in the "react-bootstrap" directory.

## Your Environment

* Version used: 5.1.2
* Browser Name and version: Chrome 80.0.3987.149
* Operating System and version (desktop or mobile): Windows 10
* React Bootstrap version: 1.4.0
* Bootstrap version: 4.5.3
* React version: 16.13.1

**Comment by ravibugude** [comment]
Yes Facing the similar issue when updated to react-bootstrap version 1.0.0

**Comment by bporter-nfuzionit** [comment]
I removed the lib directory from the index in the src and the lib directory, but then I began receiving errors with _propTypes2.default.
